core/app/assets/js/layout/header-button.js

```diff
@@ -0,0 +1,15 @@
+import BaseButton from '../ui/molecules/button';
+
+export default class Button extends BaseButton {
+	static defaultProps = Object.assign( {} /* clone */, BaseButton.defaultProps, {
+			hideText: true,
+	} );
+
+	getCssId() {
+		return `eps-app-headerBtn-` + super.getCssId();
```

Comment:
```suggestion
		return `eps-app-header-btn-` + super.getCssId();
```

Reply:
Done.
Note we need to decide what to do about class components ... we have a few of them